sds_data_manager/lambda_code/SDSCode/database_handler.py

```diff
@@ -24,7 +24,7 @@ def update_file_catalog_table(session, metadata_params):
     """
     try:
         # Add data to the file catalog table
-        session.add(models.FileCatalog(**metadata_params))
+        session.add(models.ScienceFiles(**metadata_params))
```

Comment:
Right now, it's only called by indexer lambda but we may need to add some logics in the future if this function is called by others too to make sure we update we update the correct table. Or change this function name to be `update_science_catalog_table` or something. Just a suggestions. You are better with names.

Reply:
I actually inlined the `update_file_catalog` since it was only a one-line function in that extra file. I agree though that we will need to follow up with more handling of ancillary and spice file uploads later.